cabra/src/pages/components/RelatedTask.tsx

```diff
@@ -59,8 +59,14 @@ export default function RelatedTask({
           id={task.id}
           checked={task.status !== TaskStatus.TODO}
           onChange={handleIsDoneChange}
+          disabled={task.isBlocked}
         />
       </p>
+      {task.isBlocked && (
+        <p tw="flex items-center text-coolGray-500 pt-1">
+          Task is blocked by other task.
+        </p>
+      )}
```

Comment:
Same as in `TaskListItem.tsx`.
Other checkbox design with tooltip would be better.